### DISCUSSION TITLE
Notice: Temporary Interaction Limits on New Accounts

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#48334] [Announcements]
Hi folks!

In an effort to reduce spammy behavior, we are instituting temporary interaction limits to bar accounts less than 24hrs-old from participating in the github.com/community Discussions space. This is an experiment and the experiment will run from Feb 22, 2023 to March 22, 2023.

Accounts that are at least 24hrs old will be able to post and interact as normal.

If your account is less than 24hrs old and you have a question, please try using the Discussions search bar above to see if your question has already been asked or simply wait a day. We apologize if this causes any inconvenience. 

If you'd like to learn more about implementing temporary interaction limits in your own orgs and repos, please read up on the official documentation [here](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-organization).

Best,
Your community moderation team

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
